libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/TripStatus.kt

```diff
@@ -17,5 +18,6 @@ data class TripStatus(
     val enhancedLocation: Location,
     val keyPoints: List<Location>,
     val routeProgress: RouteProgress?,
-    val offRoute: Boolean
+    val offRoute: Boolean,
```

Comment:
Could we remove all these as that data comes from the `NavigationStatus`? 🤔

Reply:
Let's do that in the follow-up refactor that I mentioned in the OP.

Reply:
Could you ticket it out so we don't forget? 🙏

Reply:
https://github.com/mapbox/mapbox-navigation-android/issues/3735